R/commonMachineLearningRegression.R

```diff
@@ -206,15 +210,18 @@
                       "knn" = gettext("K-Nearest Neighbors Regression"),
                       "regularized" = gettext("Regularized Linear Regression"),
                       "randomForest" = gettext("Random Forest Regression"),
-                      "boosting" = gettext("Boosting Regression"))
+                      "boosting" = gettext("Boosting Regression"),
```

Comment:
Maybe <kbd>Ctrl</kbd>+<kbd>I</kbd> to indent? There are a couple spots where the indentation is off (lines 199-201, 213-214, 222-224, etc.)